--- libudev0-shim-1/Makefile.org	2016-12-28 20:42:20.916359893 +0000
+++ libudev0-shim-1/Makefile	2016-12-28 22:18:26.846955815 +0000
@@ -5,10 +5,19 @@
 default: all
 
 all: \
-	libudev.so.0.0.9999
+	libudev.so.0.0.9999 libudev32.so.0.0.9999
 
 libudev.so.0.0.9999: libudev0.c
 	$(CC) $(CFLAGS) $(LDFLAGS) $(LDLIBS) $^ -o $@
 
+libudev32.so.0.0.9999: libudev0.c
+	$(CC) $(CFLAGS) -m32 $(LDFLAGS) $(LDLIBS) $^ -o $@
+	
+install: libudev.so.0.0.9999 libudev32.so.0.0.9999
+	mkdir -p $(DESTDIR)/usr/lib64
+	mkdir -p $(DESTDIR)/usr/lib32
+	cp libudev.so.0.0.9999  $(DESTDIR)/usr/lib64
+	cp libudev32.so.0.0.9999  $(DESTDIR)/usr/lib32/libudev.so.0.0.9999
+
 clean:
 	$(RM) libudev.so.0.0.9999
