diff --git a/Makefile b/Makefile
index a8d1991..0330c48 100644
--- a/Makefile
+++ b/Makefile
@@ -5,10 +5,21 @@ LDLIBS := -ludev
 default: all
 
 all: \
-	libudev.so.0.0.9999
+	libudev.so.0.0.9999 libudev32.so.0.0.9999
 
 libudev.so.0.0.9999: libudev0.c
 	$(CC) $(CFLAGS) $(LDFLAGS) $(LDLIBS) $^ -o $@
 
+libudev32.so.0.0.9999: libudev0.c
+	$(CC) $(CFLAGS) -m32 $(LDFLAGS) $(LDLIBS) $^ -o $@
+
+install: libudev.so.0.0.9999 libudev32.so.0.0.9999
+	mkdir -p $(DESTDIR)/usr/lib64
+	mkdir -p $(DESTDIR)/usr/lib32
+	cp libudev.so.0.0.9999  $(DESTDIR)/usr/lib64
+	cp libudev32.so.0.0.9999  $(DESTDIR)/usr/lib32/libudev.so.0.0.9999
+	ln -s libudev.so.0.0.9999  $(DESTDIR)/usr/lib64/libudev.so.0
+	ln -s libudev.so.0.0.9999  $(DESTDIR)/usr/lib32/libudev.so.0
+
 clean:
 	$(RM) libudev.so.0.0.9999
